hazelcast/src/main/java/com/hazelcast/cp/internal/datastructures/spi/blocking/AbstractBlockingService.java

```diff
@@ -272,6 +272,12 @@ public final RR getRegistryOrNull(CPGroupId groupId) {
         return Collections.unmodifiableCollection(registry.getLiveOperations());
     }
 
+    public int getTotalResourcesCount() {
+        return registries.values().stream()
+                .map(collection -> collection.resources.size())
+                .reduce(0, Integer::sum);
```

Comment:
This will come with unnecessary boxing / unboxing. Please replace with `.mapToInt(collection -> collection.resources.size()).sum()`